axelrod/tests/strategies/test_meta.py

```diff
@@ -248,7 +248,9 @@ def test_strategy(self):
         self.responses_test([D], [C, C, C, C], [C, C, C, C])
 
         # After long histories tit-for-tat should come into play.
-        self.responses_test([D], [C] * 101, [C] * 100 + [D])
+        letters = 'ABEFGHIJKLMNOPQRSTUVWXYZ'
```

Comment:
Could you  add a (short) comment explaining the need for the junk history. Just want it as a pointer for when we eventually get to refactoring this test file.